packages/jupyter-ai-magics/jupyter_ai_magics/providers.py

```diff
@@ -443,12 +448,17 @@ class ChatOpenAINewProvider(BaseProvider, ChatOpenAI):
     id = "openai-chat-new"
     name = "OpenAI"
     models = [
+        "gpt-3.5-turbo",
+        "gpt-3.5-turbo-16k",
+        "gpt-3.5-turbo-0301",
+        "gpt-3.5-turbo-0613",
+        "gpt-3.5-turbo-16k-0613",
         "gpt-4",
         "gpt-4-0314",
+        "gpt-4-0613",
         "gpt-4-32k",
         "gpt-4-32k-0314",
-        "gpt-3.5-turbo",
-        "gpt-3.5-turbo-0301",
+        "gpt-4-32k-0613",
```

Comment:
I tested this, and this doesn't seem to be working for me.
```
openai.error.InvalidRequestError: The model `gpt-4-32k-0613` does not exist or you do not have access to it. Learn more: https://help.openai.com/en/articles/7102672-how-can-i-access-gpt-4.
```

Reply:
yes, those 32k models are not for everyone at the moment [How to get access to gpt-4-32k?](https://community.openai.com/t/how-to-get-access-to-gpt-4-32k/104803)